robot/CMakeLists.txt

```
@@ -1,5 +1,19 @@
 find_package(Python COMPONENTS Interpreter)
 
+execute_process(
+  COMMAND ${Python_EXECUTABLE} -c "import robot"
+  RESULT_VARIABLE EXIT_CODE
+  OUTPUT_QUIET
+)
+
+if (NOT ${EXIT_CODE} EQUAL 0) 
+  message(
+    FATAL_ERROR
+    "The python package 'robotframework' could not be found"
```

Please use the active voice:

```suggestion
    "Failed to import the 'robot' Python package"
```